src/core_plugins/tile_map/public/kibana_map.js

```diff
@@ -300,7 +301,7 @@ export class KibanaMap extends EventEmitter {
     return zoomToPrecision(this._leafletMap.getZoom(), 12, this._leafletMap.getMaxZoom());
   }
 
-  getBounds() {
+  getBounds(isTrimmed = true) {
```

Comment:
consider not having the default param and create different method called `getTrimmedBounds`